migration-tool/src/main/java/software/amazon/awssdk/migration/internal/recipe/V1GetterToV2.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.migration.internal.recipe;
+
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Recipe;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.tree.Expression;
+import org.openrewrite.java.tree.J;
+import org.openrewrite.java.tree.JavaType;
+import org.openrewrite.java.tree.TypeUtils;
+import software.amazon.awssdk.annotations.SdkInternalApi;
+import software.amazon.awssdk.migration.internal.utils.NamingUtils;
+
+@SdkInternalApi
+public class V1GetterToV2 extends Recipe {
+    @Override
+    public String getDisplayName() {
+        return "V1 Getter to V2";
+    }
+
+    @Override
+    public String getDescription() {
+        return "Transforms V1 getter to fluent getter in V2.";
+    }
+
+    @Override
+    public TreeVisitor<?, ExecutionContext> getVisitor() {
+        return new V1GetterToV2Visitor();
+    }
+
+    private static class V1GetterToV2Visitor extends JavaIsoVisitor<ExecutionContext> {
+
+        @Override
+        public J.MethodInvocation visitMethodInvocation(J.MethodInvocation method, ExecutionContext executionContext) {
+            method = super.visitMethodInvocation(method, executionContext);
+
+            JavaType selectType = null;
+
+            Expression select = method.getSelect();
+            if (select != null) {
+                selectType = select.getType();
+            }
+
+            if (selectType == null) {
+                return method;
+            }
+
+            String methodName = method.getSimpleName();
+            JavaType.FullyQualified fullyQualified = TypeUtils.asFullyQualified(selectType);
+
+            if (NamingUtils.isGetter(methodName)) {
+                methodName = NamingUtils.removeGet(methodName);
```

Comment:
How to we make sure it does not remove getters which are not associated with v1 variables ?
Can we add a test cases where we have instances as below

```
Locale locale = Locale.getDefault();
String path = System.getenv("PATH");
String className = String.class.getName(); 
```

Reply:
Added check for model class and additional test cases